thunderfish/checkpulse.py

```diff
@@ -12,6 +12,85 @@
 import matplotlib.pyplot as plt
 from matplotlib.patches import Rectangle
 
+def psd_type_plot(freqs, power, proportions, percentiles, ax, fs, max_freq = 3000):
```

Comment:
unify the function name with the corresponding one from the pulse width algorithm. 

Move the function below the pusle_width function.